StatisticalInference.py

```diff
@@ -0,0 +1,62 @@
+from typing import Tuple
+import pandas as pd
+import statsmodels.stats.api as sms
+from statsmodels.stats.weightstats import ttest_ind
+
+
+class ConductStatisticalInference:
+    AB_ALPHA: float = 0.05
+    POWER: float = 0.8
+    MDE: float = 0.05
+
+    def __init__(self, ab_data: pd.DataFrame) -> None:
+        self.ab_data: pd.DataFrame = ab_data
+        self.ab_control, self.ab_treatment = self.ab_data_handler()
+        self.ab_t_stat, self.ab_p_value, self.ab_df = self.analyze_statistics()
+        self.lb, self.ub = self.confidence_interval_cal()
+
+    def ab_data_handler(self) -> Tuple[pd.Series, pd.Series]:
+        ab_control = self.ab_data[self.ab_data['GROUP'] == 0].groupby('USER_ID')['SALES'].mean()
+        ab_treatment = self.ab_data[self.ab_data['GROUP'] == 1].groupby('USER_ID')['SALES'].mean()
+        return ab_control, ab_treatment
+
+    def analyze_statistics(self) -> Tuple[float, float, float]:
+        ab_t_stat, ab_p_value, ab_df = ttest_ind(self.ab_control, self.ab_treatment)
+        return ab_t_stat, ab_p_value, ab_df
+
+    def confidence_interval_cal(self) -> Tuple[float, float]:
+        desc_stats_test = sms.DescrStatsW(self.ab_treatment)
+        desc_stats_control = sms.DescrStatsW(self.ab_control)
+        cm = sms.CompareMeans(desc_stats_test, desc_stats_control)
+        lb, ub = cm.tconfint_diff(usevar='unequal')
+        return lb, ub
+
+    def print_analysis_result(self) -> None:
```

Comment:
This function is not just printing, it is calculating things and printing. Decouple these two things that the method is doing. Have one method for the calculations, and another method for the printing